specs/commandline/inspect.md

```diff
@@ -0,0 +1,295 @@
+# notation inspect
+
+## Description
+
+Use `notation inspect` command to inspect all the signatures associated with signed artifact in a human readable format.
+
+Upon successful execution, both the digest of the signed artifact and the digests of signatures manifest along with their properties associated with the signed artifact are printed in the following format:
+
+```shell
+<registry>/<repository>@<digest>
+└── application/vnd.cncf.notary.signature
+    ├── <digest_of_signature_manifest>
```

Comment:
nit: please use camel case for all of the below fields or don't use it at all. my preference is later one.

Reply:
no camel case used anywhere in the tree view.